geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExecuteFunctionCommand.java

```diff
@@ -73,7 +72,7 @@ public ResultModel executeFunction(
       // find the members based on the groups or members
       dsMembers = findMembers(onGroups, onMembers);
     } else {
-      dsMembers = findMembersForRegion(onRegion);
+      dsMembers = findAnyMembersForRegion(onRegion);
```

Comment:
`--region` option supersedes `--group` and `--member` show there should be check here for the invalid combination.

Reply:
this is checked in the interceptor

Reply:
OK. I missed that in my first review